tools/travis-test.sh

```diff
@@ -60,7 +60,16 @@ summaries_dir() {
 }
 
 run_small_tests() {
+  echo "############################"
+  echo "Running small tests (test/)"
```

Comment:
This is printed in `myabe_run_small_tests` function which calls `run_small_tests`. There is no need to print it twice.
The only case when the script runs `run_small_tests` is for the job were only small tests are run. In this case adding `-s false` will not disable small tests anyway.